tests/0081-admin.c

```diff
@@ -2708,15 +2727,44 @@ static void do_test_DescribeConsumerGroups(const char *what,
                     i, exp->group_id,
                     rd_kafka_ConsumerGroupDescription_group_id(act));
                 if (i < known_groups) {
+                        int member_count;
+                        const rd_kafka_MemberDescription_t *member;
+                        const char *client_id;
+                        const char *group_instance_id;
```

Comment:
Please put some air into the code. Have newlines between variable declarations and the first statement.
Use newline between logical section of the code.

It makes it easier to read the code.

Reply:
done